makefiles/Makefile.cloud-docs-osb

```diff
@@ -94,27 +55,43 @@ deploy: build/public
 
 	$(MAKE) deploy-search-index
 
+#################################################################
+####                                                         ####
+####             SNOOTY MAKE TARGETS                         ####
+####                                                         ####
+#################################################################
+
 ## Update the search index for this branch
 deploy-search-index:
 	@echo "Building search index"
-	if [ ${STABLE_BRANCH} = ${GIT_BRANCH} ]; then \
-		mut-index upload build/public/${GIT_BRANCH} -o ${PROJECT}-current.json --aliases ${PROJECT}-${GIT_BRANCH} -u ${PRODUCTION_URL}/${PROJECT}/stable -g -s; \
-	else \
-		mut-index upload build/public/${GIT_BRANCH} -o ${PROJECT}-${GIT_BRANCH}.json -u ${PRODUCTION_URL}/${PROJECT}/${GIT_BRANCH} -s; \
-	fi
+	mut-index upload public -o ${PROJECT}-${GIT_BRANCH}.json -u ${PRODUCTION_URL} -s
```

Comment:
this will need to be changed to `mut-index upload public -o ${PROJECT -${GIT_BRANCH}.json -u ${PRODUCTION_URL}/atlas-open-service-broker -s`